cantabular-import/scs.sh

```diff
@@ -260,39 +317,72 @@ chown() {
     sudo chmod 755 -R ${ZEBEDEE_GENERATED_CONTENT_DIR}
 }
 
-upServices () {
-    echo "Starting dp cantabular import..."
+startDetachedServices () {
     cd "$DP_CANTABULAR_IMPORT_DIR"
+    logSuccess "Starting dp cantabular import..."
     make start-detached
-    # make start
-    echo "Starting dp cantabular import... Done."
+
+    numStartedSrv=$(docker ps | grep -v CONTAINER | wc -l)
+    attempt=0
+
+    while [ $numStartedSrv -ne $EXPECTED_RUNNING_SERVICES ] && [ $attempt -lt $NUMBER_OF_RETRIES ];  do
```

Comment:
Again not an issue for everyone, but for those who have few extra service in their local docker will have to updated the count, or have to go with checking the service by name